tests/integration/tasks/test_task_utils.py

```diff
@@ -0,0 +1,109 @@
+import logging
+import pytest
+import pandas as pd
+
+from prefect.tasks.secrets import PrefectSecret
+
+from viadot.tasks import AzureSQLCreateTable
+from viadot.exceptions import ValidationError
+from viadot.task_utils import EnsureDFColumnOrder
+
+
+SCHEMA = "sandbox"
+TABLE = "test"
+
+CREDENTIALS = PrefectSecret("AZURE_DEFAULT_SQLDB_SERVICE_PRINCIPAL_SECRET").run()
+
+
+def test_ensure_df_column_order_append_same_col_number(caplog):
+    create_table_task = AzureSQLCreateTable()
+    with caplog.at_level(logging.INFO):
+        create_table_task.run(
+            schema=SCHEMA,
+            table=TABLE,
+            dtypes={"id": "INT", "name": "VARCHAR(25)", "street": "VARCHAR(25)"},
+            if_exists="replace",
+        )
+    assert "Successfully created table sandbox" in caplog.text
```

Comment:
Small improvement - maybe is better to change the message to `Successfully created table {SCHEMA}"` - to avoid errors when a variable `SCHEMA` is changed